core/src/main/java/com/novoda/merlin/registerable/MerlinCallbackManager.java

```diff
@@ -4,14 +4,14 @@
 
 public class MerlinCallbackManager<T extends Registerable> {
 
-    private final MerlinConnector<T> merlinConnector;
+    private final Register<T> register;
```

Comment:
`MerlinConnector` was confusing because it is not connecting to anything but is a first-class collection around `Registerable`s